packages/grid/x-data-grid/src/utils/utils.ts

```diff
@@ -10,6 +10,10 @@ export function isObject(value: any): value is Record<string, any> {
   return typeof value === 'object';
 }
 
+export function isObjectEmpty(value: any): boolean {
+  return Object.keys(value).length === 0;
```

Comment:
Not important, but this function is only used inside `x-data-grid-pro`. It could be moved to a utils file inside this package, or even be inlined since the check is simple.

Reply:
Removed the helper.